ci/prow/make_config.go

```diff
@@ -496,6 +562,8 @@ func generatePresubmit(title string, repoName string, presubmitConfig yaml.MapSl
 		case "go-coverage-threshold":
 			data.Base.GoCoverageThreshold = getInt(item.Value)
 			repoData.GoCoverageThreshold = data.Base.GoCoverageThreshold
+		case "nil":
```

Comment:
This offers no clue as to what setting this governs in the yaml. Can we change it to something more clear, like "braches" (and then "legacy-branches" become just "legacy")? Actually, why the two-level setting?